From: =?utf-8?q?Jonas_Dre=C3=9Fler?= <verdre@v0yd.nl>
Date: Sat, 28 Mar 2020 14:34:24 +0100
Subject: layout: Use translation_y of 0 to hide keyboard

Since we show the keyboard using a translation_y of -keyboardHeight, the
keyboard will be moved down far enough to be out of sight by setting
translation_y to 0.

https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/1142

Applied-Upstream: 3.36.1
Origin: https://gitlab.gnome.org/GNOME/gnome-shell/commit/b68fb357
---
 js/ui/layout.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/ui/layout.js b/js/ui/layout.js
index a02cab6..67100d1 100644
--- a/js/ui/layout.js
+++ b/js/ui/layout.js
@@ -765,7 +765,7 @@ var LayoutManager = GObject.registerClass({
             this._keyboardHeightNotifyId = 0;
         }
         this.keyboardBox.ease({
-            translation_y: this.keyboardBox.height,
+            translation_y: 0,
             opacity: 0,
             duration: immediate ? 0 : KEYBOARD_ANIMATION_TIME,
             mode: Clutter.AnimationMode.EASE_IN_QUAD,
